ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-with-range-scan.json

```diff
@@ -0,0 +1,354 @@
+{
+  "comments": [
```

Comment:
We should also test some of the multi-column key cases. One way to do this is:

```
CREATE TABLE AGGREGATE AS SELECT ID1, ID2, COUNT(1) AS COUNT FROM INPUT GROUP BY ID1, ID2;
SELECT * FROM AGGREGATE WHERE ID1='11' AND ID2=10;
```

Reply:
Thanks for pointing out the multi-column cases. Got all those cases covered in the validation tests now.